IM_calculation/scripts/calculate_ims.py

```diff
@@ -163,6 +163,25 @@ def load_args():
     )
 
     args = parser.parse_args()
+
+    if constants.Components.ceas.str_value in args.components:
+        wrong_ims = ""
+        if "FAS" not in args.im:
```

Comment:
Using enum str value above, but a string here is inconsisent. 
I think this should use the enum str value